README.md

```diff
@@ -1 +1,56 @@
-# restaurant-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Project: Restaurant Page
+
+## Features
+
+- All the elements in the DOM are added using Javascript
+
+
+## Screenshots
+
+![screenshot](screenshot.png)
+
+
+## Built With
+
+- HTML5
+- CSS3
+- Javascript (ES6)
+
+## Live Demo
+
+Visit the [Restaurant Page](https://flamboyant-ride-be0a79.netlify.app/)
+
+## Prerequisites
+- Internet browser compatible with HTML5, CSS3 and Javascript ES6
+
+## Getting Started
```

Comment:
- Add instruction for installing packages, building using webpack and starting the local server